Cargo.lock

```diff
@@ -2616,7 +2618,7 @@ dependencies = [
  "diesel_derives",
  "num-bigint 0.2.6",
  "num-integer",
- "num-traits 0.2.15",
+ "num-traits 0.1.43",
```

Comment:
Running `cargo x generate-workspace-hack` changed this version, which seems to have most recently changed in #1345. I'm not sure if this is ok. Any thoughts @JoshLind @zekun000 ?